core/templates/dev/head/pages/base.html

```diff
@@ -159,6 +159,7 @@
             <noscript>
               <div class="oppia-page-cards-container">
                 <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <!-- Note to developers: Do *not* remove the replicated text because the translation engine requires JavaScript to be enabled.-->
```

Comment:
Hm, not clear enough, I think (it doesn't quite explain that the translation engine won't kick in, and it's easy to miss that this is in a noscript tag). How about:

```
<!-- Note to developers: We replicate the translated text inline because, without JavaScript enabled, the translation engine doesn't kick in. -->
```